### ISSUE TITLE
Set "AllowUserPorts: 80 443" as default in acng.conf

### ISSUE THREAD

**Issue by reefland** [sameersbn/docker-apt-cacher-ng#39]
Since the "acng.conf" is not exposed externally nor changes persisted.  Would like to set `AllowUserPorts: 80 443` in order address a bug introduced in apt-cacher-ng 3.74:

<https://bugs.launchpad.net/ubuntu/+source/apt-cacher-ng/+bug/1993026>

This bug prevents `https` caching when using the "tell-me-what-you-need method" for  [Access to SSL/TLS remotes](https://www.unix-ag.uni-kl.de/~bloch/acng/html/howtos.html#ssluse) on client side configuration.

```
deb http://HTTPS///get.docker.com/ubuntu docker main
# If apt-cacher-ng is configured as proxy in APT, this makes it
# switch internally to https://get.docker.com/ubuntu
deb http://acnghost:3142/HTTPS///get.docker.com/ubuntu docker main
# Basically the same just with access to apt-cacher-ng through
# URL rewritting instead of setting http proxy.
```
 #38 would help with this as well.

**Comment by TheOneDoc** [comment]
It's 2025 and this is still not implemented. 
Just wasted an hour because of it.

**Comment by unnamedsos** [comment]
according to the acng.conf
```
 # A place to look for additional configuration and resource files if they are not
# found in the configuration directory
#
SupportDir: /usr/lib/apt-cacher-ng
```
and 
```
more zz_debconf.conf 
# This is a configuration generated file managed by the package configuration
# scripts of apt-cacher-ng. All manual changes here will be overriden.
# To reconfigure particular settings, please run:
# dpkg-reconfigure apt-cacher-ng -plow

# See acng.conf and apt-cacher-ng documentation for more details.
# To override this values permanently, put the assignments into a file
# like /etc/apt-cacher-ng/zzz_override.conf .

``` 
what worked for me ->
in docker compose style->
 ```  
 command: >
      bash -c "
      echo 'AllowUserPorts: 80 443'  >> /etc/apt-cacher-ng/zzz-custom.conf &&
      echo 'PassThroughPattern: ^.*:443$'  >> /etc/apt-cacher-ng/zzz-custom.conf &&
     exec /sbin/entrypoint.sh"
```
or mount it to the SupportDir: /usr/lib/apt-cacher-ng. Just make sure you use zzz since the application load order is alphabetical.
